zenodo/modules/profiles/templates/zenodo_profiles/profile.html

```diff
@@ -0,0 +1,150 @@
+            <img width="26px" style="vertical-align:middle; margin-left: 5px" src="{{ url_for('static', filename='img/orcid.png') }}" alt="Orcid" />
+            <a class="no_hover" href="https://orcid.org/{{ account.extra_data.get('orcid') }}">
+              <span style="margin-left: 3px;font-size: 16px">{{ account.extra_data.get('orcid') }}</span>
+            </a>
+          </div>
+          {%- endif %}
+
+          {%- if account.client_id == config.GITHUB_APP_CREDENTIALS.consumer_key %}
+          <div>
+            <i style="vertical-align:middle" class="fa fa-github fa-2x fa-fw" aria-hidden="true"></i>
+            <a class="no_hover" href="https://github.com/{{ account.extra_data.get('login') }}">
+              <span style="font-size: 16px">{{ account.extra_data.get('login') }}</span>
+            </a>
+          </div>
+          {%- endif %}
+      </div>
+      {%- endfor %}
+      {%- endif %}
+
+      {%- if current_user.id != user.id %}
+        {%- if user.researcher_profile.allow_contact_owner %}
+        <div style="margin-top:18px">
+          <a class="no_hover" href="/profile/{{ user.id }}/contact">
+            <button type="button" class="btn btn-default btn-lg btn-block">Contact</button>
+          </a>
+        </div>
+        {%- endif %}
+      {%- else %}
+      <div style="margin-top:18px">
+        <a class="no_hover" href="/account/settings/profile">
+          <button type="button" class="btn btn-default">Edit Profile</button>
+        </a>
+      </div>
+      {%- endif %}
+
+      {%- if user.researcher_profile.bio %}
+      <div>
+        <h3><small>{{ user.researcher_profile.bio }}</small></h3>
+      </div>
+      {%- endif %}
+
+      {%- if user.researcher_profile.location or user.researcher_profile.website %}
+      <hr />
+      <div style="font-size:17px">
+        {%- if user.researcher_profile.location %}
+        <div>
+          <span class="fa fa-map-marker fa-fw" aria-hidden="true"></span>
+          <span>{{ user.researcher_profile.location }}</span>
+        </div>
+        {%- endif %}
+
+        {%- if user.researcher_profile.website %}
+        <div>
+          <span class="fa fa-link fa-fw" aria-hidden="true"></span>
+          <span>
+            <a href="{{ user.researcher_profile.website }}">
+              {{ user.researcher_profile.website }}
+            </a>
+          </span>
+        </div>
+        {%- endif %}
+      </div>
+      {%- endif %}
+
+      {%- if user.communities %}
+      <hr />
+      <div>
+        <h4 style="font-size:22px; margin-bottom:0"><strong>Communities</strong></h4>
+        <ul style="padding: 18px">
+          {%- for org in user.communities %}
+          <li><h3 style="margin:0">
+            <small><a href="/communities/{{ org.id }}">{{ org.title }}</a></small>
+          </h3></li>
+          {%- endfor %}
+        </ul>
+      </div>
+      {%- endif %}
+    </div>
+
+    <div class="col-sm-9 col-xs-12">
+      <h1>Popular Uploads</h1>
```

Comment:
We decided to rename it to just "Uploads" since this is a bit inaccurate (these are not the "most viewed" or "most popular" records in any way)